src/Microsoft.AspNetCore.Http.Extensions/UriHelper.cs

```diff
@@ -101,7 +101,7 @@ public static string Encode(Uri uri)
         /// <returns></returns>
         public static string GetEncodedUrl(this HttpRequest request)
```

Comment:
Super odd to have an extension method in this class. Also, any reason the above `Encode` method shouldn't be an extension method/renamed `GetEncodedUrl`?